scripts/react-native-xcode.sh

```diff
@@ -194,14 +194,12 @@ else
   if [[ $EMIT_SOURCEMAP == true ]]; then
     EXTRA_COMPILER_ARGS="$EXTRA_COMPILER_ARGS -output-source-map"
   fi
-  HBC_FILE="$CONFIGURATION_BUILD_DIR/$(basename $BUNDLE_FILE)"
-  "$HERMES_CLI" -emit-binary $EXTRA_COMPILER_ARGS -out "$HBC_FILE" "$BUNDLE_FILE"
-  mv "$HBC_FILE" "$DEST/"
-  BUNDLE_FILE="$DEST/main.jsbundle"
+  "$HERMES_CLI" -emit-binary $EXTRA_COMPILER_ARGS -out "$DEST/main.jsbundle" "$BUNDLE_FILE" 
```

Comment:
Decided to remove `mv` stage and write directly into `$DEST`.

Reply:
Such a maverick! 🤠 

👍